incident.go

```diff
@@ -657,15 +657,20 @@ type ResponderRequestResponse struct {
 // ResponderRequestTarget specifies an individual target for the responder request.
 type ResponderRequestTarget struct {
 	APIObject
-	Responders IncidentResponders `json:"incident_responders"`
+	Responders IncidentResponders `json:"incident_responders,omitempty"`
+}
+
+// ResponderRequestTargets is a wrapper for a ResponderRequestTarget.
+type ResponderRequestTargets struct {
```

Comment:
I know this is a little strange but could we call this something like `ResponderRequestTargetWrapper`? Calling it `ResponderRequestTargets` [plural] is a little strange because it isn't actually a list or collection of any kind...

Reply:
@ChuckCrawford Thanks so much for taking a look at this!  And I totally agree - `ResponderRequestTargetWrapper` is both clearer and more accurate (I had noticed that false pluralization as well, but I chickened out of changing the name and just left it as it was named in its last working state).  Edit made, and squashed the commits to keep the history tidier.